modules/category/category.graphql.js

```diff
@@ -12,7 +12,7 @@ const categoryType = `
 
 const categoryInput = `
     input CategoryInput {
-    categoryCode: String
+    code: String
     name: [LanguageInput]
     images: ImageSetInput
     subcategories: [ID]
```

Comment:
I thought that when we create a category we are going to provide subcategories as an array of objects with category data

Reply:
The UI would become more complicated if we were to create subcategories simultaneously with main category, like more tabs embedded into other tabs. I think creating category first and then subcategories after would be much simpler.